xmtp_mls/src/owner/evm_owner.rs

```diff
@@ -1,16 +0,0 @@
-pub use ethers::signers::{LocalWallet, Signer};
-
-use xmtp_cryptography::signature::{h160addr_to_string, RecoverableSignature, SignatureError};
-
-use crate::InboxOwner;
-
-impl InboxOwner for LocalWallet {
```

Comment:
I preferred the implementation in `xmtp_id`, enabling us to remove the `ethers` dependency in `xmtp_mls`